infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -212,6 +216,8 @@ private void trim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext ctx, I
      */
     Thread gcThread;
 
+    Thread writerThread;
```

Comment:
I wasn't using any of the executor service functionality, I'll change it anyways.